python/scrutiny/server/device/device_handler.py

```diff
@@ -87,15 +92,28 @@ def __init__(self, config, datastore):
         self.heartbeat_generator.set_interval(max(0.5, self.config['heartbeat_timeout'] * 0.75))
         self.comm_broken = False
         self.device_id = None
+        self.operating_mode = self.OperatingMode.Normal
 
         self.reset_comm()
 
+    def set_operating_mode(self, mode):
```

Comment:
```suggestion
    def set_operating_mode(self, mode: DeviceHandler.OperatingMode) -> None:
```

Reply:
You did teach me sensei.

Reply:
There will be a PR only for type checking in the future. Thanks